src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperOutput.cs

```diff
@@ -33,9 +35,11 @@ internal TagHelperOutput(string tagName)
         {
             TagName = tagName;
             Attributes = new Dictionary<string, object>(attributes, StringComparer.OrdinalIgnoreCase);
+            _preElement = new DefaultTagHelperContent();
```

Comment:
We could use property initializers for these now - `public TagHelperContent PreElement { get; } = new DefaultTagHelperContent();`